doc/answerfile.txt

```diff
@@ -46,7 +46,8 @@ Common Elements
     nfs
 
   'source' defines the location of the installation repository or a
-  Supplemental Pack. There may be multiple 'source' elements.
+  Supplemental Pack. There may be multiple 'source' elements. It is
```

Comment:
Would it be better to say: "The may be zero or multiple 'source' elements."?

Reply:
The problem is, for `<installation>` elements, it is mandatory, I believe insisting on the "zero" case which is only for `<restore>` can only add confusion.  I also realized `<bootloader>` and `<network-backend>`, also handled in `parseCommon()` are in the same situation: in fact the "Common elements" section relates to "Sub-elements for 'installation', common to all three modes".  So likely this section should be split in two, with `driver-source`, `fcoe-interface' and `*script` split as being the real "common elements".